OJP/OJP_Trips.xsd

```diff
@@ -878,6 +895,172 @@
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="low"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="EnvironmentalSafety" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference about climate/environmental-friendliness of the trip (option 'low' might have no effect).</xs:documentation>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="high"/>
+						<xs:enumeration value="average"/>
+						<xs:enumeration value="low"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:element name="WeatherProtection" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference not be exposed to eventual weather hazards, thus, to avoid walks and micromobility.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ExtraSafe" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for an extra high level of safety.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ExtraReliable" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for an extra high reliability (low probabilty of delays, cancellations, etc.).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ExtraAccessible" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for perfect accessibility (lifts, ramps, no stairs, etc.), including extra transfer time, short transfer distances, etc.. May be superseeded by EPIAP updates. TODO</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Scenic" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for scenic, touristic impressions on some parts of the trip.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Peaceful" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference for peacful transport (quiet travel compartements or low occupation).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="TripReason" type="TripReasonEnumeration" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>A classification of a passenger's reason for undertaking a TRIP PATTERN (TRIP REASON from Transmodel).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="WalkingProfile" default="normal" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>Users walking profile (especially for hiking)</xs:documentation>
+				</xs:annotation>
```

Comment:
May be somewhat exotic or rarely used preference. Is it useful in the context of the other user preferences? Are these colour codes standardized in Europe? Most countries will have none of these.